com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/ObjectStoreFileStorage.java

```diff
@@ -43,15 +44,14 @@ public ObjectStoreFileStorage(BlobStore blobStore, String container) {
     }
 
     @Override
-    public void addFile(FileEntry fileEntry, InputStream fileStream) throws FileStorageException {
+    public void addFile(FileEntry fileEntry, File file) throws FileStorageException {
         String entryName = fileEntry.getId();
         long fileSize = fileEntry.getSize()
             .longValue();
         Blob blob = blobStore.blobBuilder(entryName)
-            .payload(fileStream)
+            .payload(file)
             .contentDisposition(fileEntry.getName())
             .contentType(MediaType.OCTET_STREAM.toString())
-            .contentLength(fileSize)

```

Comment:
this is automatically set by BlobBuilder.payload(File file) so it's no longer needed.